covid-mapper/features/drawer/DrawerContent.js

```diff
@@ -58,22 +70,22 @@ const DrawerContent = (props) => {
       <DrawerContentScrollView {...props}>
         <LogoContainer>
           {/* Logo */}
-          <LogoImage source={require('../../assets/logo.png')}/>
+          <LogoImage source={require("../../assets/logo.png")} />
           <LogoText>COVID Mapper</LogoText>
         </LogoContainer>
         <GreenBorder></GreenBorder>
         {/*End Logo section and Begin DrawerItems section */}
         <DrawerSection>
           {/* World total section */}
           <Heading>World</Heading>
-         
+          {/* endpoint: /v3/covid-19/all AND /v3/covid-19/countries */}
```

Comment:
This comment still needed?
I just saw all of the other commented out code. I think they are all supposed to be there